docs/reference/kernel_language.rst

```diff
@@ -1327,8 +1357,7 @@ To read a high-resolution timer from the device, HIP provides the following buil
 
   .. code:: cpp
 
-    clock_t clock()
-    long long int clock64()
+    long long int wall_clock64()
```

Comment:
They are still 
__device__ long long int clock64();
__device__ long long int clock();

But we need note that clock() and clock64() do not work normally in Gfx11.

Reply:
Updated